### PR TITLE
[zh-cn] sync administer-cluster/memory-manager.md

### PR DESCRIPTION
**Pull request by my-git9**
- sync administer-cluster/memory-manager.md
- add  anchors

`content/zh-cn/docs/tasks/administer-cluster/memory-manager.md`

### CODE REVIEW

**Review by howieyuen — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/tasks/administer-cluster/memory-manager.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ in order to enable the Memory Manager feature.
 <!--
 ## How Memory Manager Operates?
 -->
-## 内存管理器如何运作？
+## 内存管理器如何运作？  {#how-memory-manager-operates}
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/memory-manager.md:91]: Do we have links to this section?

Comment (my-git9) [reply content/zh-cn/docs/tasks/administer-cluster/memory-manager.md:91]: Please see.
https://kubernetes.io/docs/tasks/administer-cluster/memory-manager/#how-memory-manager-operates
https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/memory-manager/#%E5%86%85%E5%AD%98%E7%AE%A1%E7%90%86%E5%99%A8%E5%A6%82%E4%BD%95%E8%BF%90%E4%BD%9C

Comment (tengqm) [reply content/zh-cn/docs/tasks/administer-cluster/memory-manager.md:91]: I mean ... are there any existing doc pages pointing to this section?

Comment (my-git9) [reply content/zh-cn/docs/tasks/administer-cluster/memory-manager.md:91]: Seems no

Comment (tengqm) [reply content/zh-cn/docs/tasks/administer-cluster/memory-manager.md:91]: So ... I would suggest we don't bother to add these anchors, unless we believe there could be links pointing to such a heading.

Comment (my-git9) [reply content/zh-cn/docs/tasks/administer-cluster/memory-manager.md:91]: ok

**Pull request closed without merging**